pkg/util/util.go

```diff
@@ -1,7 +1,17 @@
 package util
 
 type StorageClassEnforcement struct {
-	AllowList []string `yaml:"allowList"`
-	AllowAll bool `yaml:"allowAll"`
-	AllowDefault bool `yaml:"allowDefault"`
-}
\ No newline at end of file
+	AllowList    []string `yaml:"allowList"`
+	AllowAll     bool     `yaml:"allowAll"`
+	AllowDefault bool     `yaml:"allowDefault"`
+}
+
+// Contains tells whether a contains x.
+func Contains(arr []string, val string) bool {
```

Comment:
Go 1.21 has slices.Contains - https://go.dev/doc/go1.21#slices if you are interested

Reply:
We can look at that when we move to using go 1.21 for the build. Depending on what builder container we use, we might already do that.